CHANGELOG.md

```diff
@@ -29,7 +29,7 @@ These changes are available in the [master branch](https://github.com/PrefectHQ/
 
 ### Breaking Changes
 
-- None
+- Remove `env_var` initialization from `EnvVarSecret` in favor of `name` - [#2302](https://github.com/PrefectHQ/prefect/pull/2302)
```

Comment:
Note that I don't believe anyone is actually using this task, as it's more of a nice example than a truly useful thing.